src/windows/MediaProxy.js

```diff
@@ -159,7 +176,21 @@ module.exports = {
         var id = args[0];
         var src = args[1];
 
-        var normalizedSrc = src.replace(/\//g, '\\');
+        src = setTemporaryFsByDefault(src);
+
+        var success = function () {
+            Media.onStatus(id, Media.MEDIA_STATE, Media.MEDIA_RUNNING);
+            win();
+        };
+
+        var error = function (reason) {
+            Media.onStatus(id, Media.MEDIA_ERROR, reason);
+            lose(reason);
+        };
+
+        // Fixing paths like: ms-appdata:///temp//recs/memos/media.m4a => ms-appdata:///temp/recs/memos/media.m4a
+        var normalizedSrc = src.replace(/([^\/])(\/\/)([^\/])/g, '$1/$3');
```

Comment:
This is another instance in which using Windows.Foundation.Uri may be preferable.  You can canonicalize the Uri using it, instead of worrying about doing accurate regular expression matching.  See for example https://msdn.microsoft.com/en-us/library/windows/apps/windows.foundation.uri.absolutecanonicaluri.aspx